### ISSUE TITLE
Consider moving wgpu helpers to separate module

### ISSUE THREAD

**Issue by kazimuth** [nannou-org/nannou#663]
Currently, `nannou` reexports a modified version of `wgpu` with a bunch of extra features added, including renaming some `wgpu` structs. I think this could be confusing for someone e.g. reading the `wgpu` documentation. It's definitely been confusing to me while working on the code for #655 :)

Since #655 involves a bunch of breaking changes for the `wgpu` API anyway, I think (after #655 is merged) it might be reasonable to do another PR after it's merged that changes nannou's `wgpu` reexport to be just `wgpu::*`, and then make a separate module with helper functionality.
